.github/workflows/qmd_tohtml.yml

```diff
@@ -0,0 +1,42 @@
+name: Convert qmd to html
+
+on:
+  push:
+
+jobs:
+  convert:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          pip install quarto
```

Comment:
This installs a python interface to quarto, and not quarto itself. If you want quarto itself, check out something like https://github.com/quarto-dev/quarto-actions/tree/main/setup.

Reply:
Thanks, I will check the Github action!